front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -125,22 +121,15 @@ const IdeasShowPage = () => {
             compact={isSmallerThanTablet}
           />
         </Box>
-        <Box
-          position="fixed"
-          top={
-            isSmallerThanTablet
-              ? undefined
-              : stylingConsts.menuHeight.toString() // Show CTA at top of screen on desktop
-          }
-          bottom={isSmallerThanTablet ? '0px' : undefined} // Show CTA at bottom of screen on mobile
-          width="100vw"
-          zIndex="1000"
-        >
-          {project?.data.id && // Show the CTA bar only when voting is active
-            isIdeaInCurrentPhase &&
-            participationContext?.attributes.participation_method ===
-              'voting' && <ProjectCTABar projectId={project.data.id} />}
```

Comment:
Found these conditions also a bit hard to understand. project id check could be removed by introducing the project check, the other vars were able to use the more descriptive var.